lib/galaxy/datatypes/sequence.py

```diff
@@ -818,9 +821,24 @@ class FastqSanger(Fastq):
     @staticmethod
     def quality_check(lines):
         """Presuming lines are lines from a fastq file, return True if the qualities are compatible with sanger encoding"""
+        is_ambiguous = True
+        within_upper_bound = True
         for line in islice(lines, 3, None, 4):
-            if not all(_ >= '!' and _ <= 'M' for _ in line[0]) or ' ' in line:
+            if ' ' in line:
```

Comment:
```suggestion
            line0 = line[0]
            if ' ' in line0:
```

Reply:
Ah, that is an interesting point! I have to say I had no clue why @mvdbeek added the `or ' ' in line` in commit https://github.com/galaxyproject/galaxy/commit/a151b63a622424c1f5f21f97169121b4c7bcc5d5 so I thought I better just stick with it. I don't think this is supposed to check for presence of a space in the quality string since `' ' < '!'` anyway.

Reply:
Well, `lines` is an array of arrays that comes from `iter_headers(file_prefix, sep='\n', count=1000)`, so each line should contain a single element.

Reply:
Yeah, I figured this out after a while, but why would you want to check that this single element is the string `' '`?

Reply:
I think we haven't resolved the doubt about the `' ' in line` check yet: should it be left as it is (I think it's wrong), changed to `' ' in line[0]` or dropped altogether?

Reply:
Looks like it probably didn't do what it was supposed to do either way and no one complained, so let's drop it.